### ISSUE TITLE
Cell showing "More" multiple times

### ISSUE THREAD

**Issue by hmoule** [acro5piano/react-native-big-calendar#442]
![Simulator Screen Shot - iPhone 11 - 2021-06-10 at 16 38 50](https://user-images.githubusercontent.com/57354264/121555101-a1533280-ca0a-11eb-8d6c-1754dd9d6e0c.png)

Shouldn't it just be showing "5 more" and "7 more" once? 

Here's my code

```javascript
import React, { FunctionComponent, useMemo } from 'react';
import { ScrollView, Dimensions } from 'react-native';
import { Calendar, ICalendarEvent } from 'react-native-big-calendar';
import moment from 'moment';
import { useSelector } from 'react-redux';
import Navbar from '../commons/Navbar';
import { RootState } from '../../reducers';

const { height } = Dimensions.get('window');

const CalendarView: FunctionComponent = () => {
  const { appointments } = useSelector((state: RootState) => state.appointments);
  const events: ICalendarEvent[] = useMemo(() => {
    return appointments.map(a => {
      return { start: moment(a.Times.Start).toDate(), end: moment(a.Times.End).toDate(), title: a.Client.Forename };
    });
  }, [appointments]);
  return (
    <>
      <Navbar back />
      <ScrollView>
        <Calendar
          onPressCell={date => console.log(date)}
          onPressEvent={event => console.log(event)}
          events={events}
          height={height - 100}
          mode="month"
        />
      </ScrollView>
    </>
  );
};

export default CalendarView;
```

**Comment by acro5piano** [comment]
I'll check it weekend.

**Comment by ThangKillah** [comment]
i have same problem @@ need hot fix this

**Comment by acro5piano** [comment]
Fixed by #466 , released on v1.0.3
